dynamo-formatters/src/main/scala/uk/gov/nationalarchives/dynamoformatters/DynamoFormatters.scala

```diff
@@ -199,7 +199,7 @@ object DynamoFormatters {
 
   case class Identifier(identifierName: String, value: String)
 
-  case class FilesTablePartitionKey(id: UUID)
+  case class FilesTablePartitionKey(id: UUID, batchId: String)
```

Comment:
1. Should this be renamed to `FilesTablePrimaryKey`?
2. I could be wrong, but from what I understand, the `batchId` will be the "sort key", right? I guess for clarity/correctness, should we do this :

```
case class FilesTablePartitionKey(id: UUID)
case class FilesTableSortKey(batchId: String)

case class FilesTablePrimaryKey(FilesTablePartitionKey, FilesTableSortKey)

```

Reply:
1. Ok. I've renamed the class. 
2. That makes sense. I've changed it to `case class FilesTablePrimaryKey(partitionKey: FilesTablePartitionKey, sortKey: FilesTableSortKey)` and updated the `DynamoFormatter` so that it reads and writes it in the `{"batchId":{"S":"TEST-ID"},"id":{"S":"assetId"}}`